src/js/components/award/SummaryBar.jsx

```diff
@@ -4,21 +4,52 @@
  **/
 
 import React from 'react';
+import moment from 'moment';
 import * as Icons from '../sharedComponents/icons/Icons';
+import InfoSnippet from './InfoSnippet';
 
 const propTypes = {
-    selectedAward: React.PropTypes.object,
-    awardStatus: React.PropTypes.string
+    selectedAward: React.PropTypes.object
 };
 
 export default class SummaryBar extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.getStatus = this.getStatus.bind(this);
+    }
+
+    componentWillMount() {
+        this.getStatus(this.props.selectedAward);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.selectedAward !== this.props.selectedAward) {
+            this.getStatus(nextProps.selectedAward);
+        }
+    }
+
+    getStatus(award) {
+        const awardStart = moment(award.period_of_performance_start_date, 'MM-DD-YYYY');
+        const awardEnd = moment(award.period_of_performance_current_end_date, 'MM-DD-YYYY');
+        const current = moment();
+        let progress = "";
+        if (current.isBefore(awardStart)) {
+            progress = "Awarded";
+        }
+        else if (current.isAfter(awardEnd)) {
```

Comment:
What happens on the last day of an award? Does `isAfter()` track to midnight (that is; during the day of the last award, it counts as _after_ because midnight has passed but the day is not yet over) or does it account for the passage of the full day (isAfter 11:59:59 PM)?

In the first case, this would cause awards to show as Complete during their last day instead of In Progress (unless that is desired behavior).